### PR TITLE
Add `.setSolidColor()` method for macOS

### PR DESCRIPTION
**Pull request by codetheweb**
Closes https://github.com/sindresorhus/wallpaper/issues/66

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,8 @@
 		"node": ">=8"
 	},
 	"scripts": {
-		"test": "xo && ava && tsd"
+		"ava": "ava -s",
```
Comment (sindresorhus) [inline package.json:16]: I prefer using the AVA config in package.json to set this instead of command-line flag.

File: index.d.ts [hunk 2]
```diff
@@ -88,6 +88,21 @@ declare const wallpaper: {
 	*/
 	screens?(): Promise<string[]>;
 
+	/**
+	__macOS only.__
```
Comment (sindresorhus) [inline index.d.ts:92]:
```suggestion
	__macOS only__

```

File: readme.md [hunk 3]
```diff
@@ -88,6 +88,34 @@ Returns a `Promise<string[]>` with the available screens.
...
 
+### .setSolidColor(color, options?) *(macOS only)*
+
+Returns a `Promise`.
+
+#### color
+
+Type: `string`
+
+The color to set.
```
Comment (sindresorhus) [inline readme.md:99]: Should explicitly mention the color format.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

**Comment by codetheweb** [comment]
Thanks, should be good now.

**Comment by sindresorhus** [comment]
Thanks :)

**Comment by sindresorhus** [comment]
Interested in adding this to the CLI too? https://github.com/sindresorhus/wallpaper-cli

**Comment by codetheweb** [comment]
Done 👍 

https://github.com/sindresorhus/wallpaper-cli/pull/9

**Pull request merged**